### DISCUSSION TITLE
some problems about yarprobotinterface

### DISCUSSION THREAD

**Discussion by Yujia-Zhang0913** [robotology/community#669] [Q&A] [answered]
Hi
I have problems in initializing the robot. Actually, I am not using the joint of r_thumb_distal. But now when it initializes, it does not move and I can hear the sound of motor.If I manually put the thumb into initializing position, it can complete the initializing procedure. Do you know what is the problem with the thumb? Really thanks.

**Comment by pattacini** [comment, marked as answer]
Hi @Yujia-Zhang0913 

Probably, a better place where to ask for help is our [HW-specific ticketing system](https://github.com/robotology/icub-tech-support/issues).

**Reply by pattacini** [reply]
While opening the new ticket, @Yujia-Zhang0913, you may consider posting pictures of the thumb that will help identify the problem, which is likely due to a broken/snapped tendon.

**Reply by pattacini** [reply]
XREF: https://github.com/robotology/icub-tech-support/issues/1687.
